GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/Garage/GarageController.cs

```diff
@@ -20,18 +20,22 @@
 /// Controller for the tab/page switch in the garage.
 /// It switches pages and tabs when different tabs are clicked;
 /// It updates the coin text indicator in the garage pages.
+/// It listens to the restore purchase button.
 /// </summary>
 public class GarageController : MonoBehaviour
 {
     public GameObject tab;
     public GameObject carPage;
     public GameObject backgroundPage;
+    public GameObject restorePurchaseConfirmPanel;
+    public GameObject restorePurchaseSuccessText;
     public Text coinsCount;
 
     private const int UnselectedTabIndex = 0;
     private const int SelectedTabIndex = 1;
     private const int CarGaragePageTabIndex = 0;
     private const int BackGroundGaragePageTabIndex = 1;
+    private const float HideRestorePurchaseSuccessTextTimeOut = 5f;
```

Comment:
nit: usually when we name variables representing time, we add Sec or Milli as suffix for clarity, in your case it would be HideRestorePurchaseSuccessTextTimeOutSec

Reply:
sg